### DISCUSSION TITLE
It seems like neovis is not managed.

### DISCUSSION THREAD

**Discussion by garik-pilot** [neo4j-contrib/neovis.js#325] [General]
I reviewed the answers to the questions raised in discussion, the last answered questions have been dated in last year. Looking for the alternative of neovis that more actively supported...

**Comment by thebestnom** [comment]
The support is ok, thank you 😄
Most people just don't use the discussion, and just open issue and you can easily see that a lot of issues did got closed in the last 2 month

**Comment by garik-pilot** [comment]
Great. Sorry. I was not expecting that you are monitoring the questions,
some of them are not answered. Thanks for your help!
